api/tests/Entity/CampTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Tests\Entity;
+
+use App\Entity\Camp;
+use App\Entity\Category;
+use App\Entity\MaterialList;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ */
+class CampTest extends TestCase {
+    private Camp $campPrototype;
+    private Category $categoryPrototype;
+    private MaterialList $materialListPrototype;
+
+    public function setUp(): void {
+        $this->campPrototype = new Camp();
+        $this->campPrototype->isPrototype = true;
+        $this->campPrototype->name = 'camp-name';
+        $this->campPrototype->title = 'camp-title';
+        $this->campPrototype->motto = 'camp-motto';
+
+        $this->categoryPrototype = new Category();
+        $this->categoryPrototype->name = 'category-name';
+        $this->categoryPrototype->color = '#FF00FF';
+        $this->categoryPrototype->numberingStyle = 'a';
+        $this->campPrototype->addCategory($this->categoryPrototype);
+
+        $this->materialListPrototype = new MaterialList();
+        $this->materialListPrototype->name = 'list-name';
+        $this->campPrototype->addMaterialList($this->materialListPrototype);
+    }
+
+    public function testCopyFromPrototype() {
```

Comment:
Ich denke entweder sollten wir diesen Testcase noch weiter ausbauen, oder wir müssten die Unittests der einzelnen Entities noch anpassen. Wir haben jetzt relativ viel neue Logik, die nicht getestet ist.

Der Fehler in der Storyboard->copyFromPrototype() müsste eigentlich schon über einen Test auffliegen.